packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -449,47 +474,68 @@ const DataSeriesFormItemModal = ({
               />
             </div>
           )}
-        </div>
 
-        {(type === CARD_TYPES.VALUE || type === CARD_TYPES.IMAGE) && (
-          <div className={`${baseClassName}--input-group`}>
+          {hasTooltip && (
```

Comment:
I would suggest we keep the previous validation, by just deleting that, we can impact others that use this component, and maybe add `deprecation` if we are removing the `type` prop

Reply:
I have deleted here. But in the parent component ,I kept it same. So functionality wise there is no change. 
![image](https://github.com/carbon-design-system/carbon-addons-iot-react/assets/18572477/d7d684ff-bde0-40a1-937a-ce993bbb3975)